src/game/delete-round.ts

```diff
@@ -14,6 +14,30 @@ export class DeleteRound extends Round {
         super();
     }
 
+    private async getColumnOffset(game: IGame) {
+        const isDefindedUserOffset = await game.nvim.eval(
+            'exists("vim_be_good_delete_me_offset")',
+        );
+        let offset;
+        console.log(
+            "delete-round#getColumnOffset - isDefindedUserOffset ",
+            isDefindedUserOffset,
+        );
+        if (game.difficulty === "noob") {
+            offset = 3;
+        }
+
+        if (isDefindedUserOffset) {
+            offset = Number(
+                await game.nvim.getVar("vim_be_good_delete_me_offset"),
+            );
+            console.log("delete-round#getColumnOffset - userOffset ", offset);
+        } else {
+            offset = Math.floor(Math.random() * (40 - 5)) + 5;
```

Comment:
@ThePrimeagen @FL3SH this line will overwrite the value set on line 27, won't it?

Reply:
aaa, my bad, of course `offset = 3` will always be overwritten ;/
Tbh I personally like it way more with random offset event in 'noob' this is why I missed it.

Fixed in #70